blankiety_galantow/resources/js/main.js

```diff
@@ -9,32 +24,5 @@ const app = new Vue({
         username: function(newName) {
             window.localStorage.setItem("username", this.username);
```

Comment:
Można wykorzystać parametr funkcji.
```suggestion
            window.localStorage.setItem("username", newName);
```